coverflow.html

```diff
@@ -281,153 +281,108 @@
-        tags: ["Grades 9-12", "Java"],
-        gradeBand: "High School",
-        projectName: "Word Builder Challenge",
-        projectType: "Guided Project",
+          "https://uploads-ssl.webflow.com/6047d44fad4ed2d97b3a7a18/605162f74818653bf84bfb18_Web%20Dev.gif",
+        title: "Web Development",
+        description: "",
+        tags: ["Grades 6-10", "HTML/CSS/JS"],
+        gradeBand: "Middle-High School",
+        projectName: "",
+        projectType: "",
         video:
-          "https://res.cloudinary.com/dnx5si27b/video/upload/v1619531468/coverflow/Build_a_Word_v1_bphziq.mp4",
+          "",
         standards: [
-          "AP CS A: MOD-1.D.1",
-          "AP CS A: CON-1.B.1",
-          "AP CS A: CON-2.A.2",
-          "AP CS A: VAR-1.E.3",
+          "",
         ],
       },
       {
         image:
-          "https://uploads-ssl.webflow.com/6047d44fad4ed2d97b3a7a18/605162f80e5dc1cacf0268ea_AP%20CS%20A-min.gif",
-        title: "AP Computer Science A",
-        description:
-          "Students create a virtual game designed to simulate a real-life chess match between two players. Each player chooses their side and the program’s algorithms set players up to adhere to the many complex and interconnected rules of chess.",
-        tags: ["Grades 9-12", "Java"],
-        gradeBand: "High School",
-        projectName: "Chess Game",
-        projectType: "Guided Project",
+          "https://storage.googleapis.com/hw-assets-dev/courses/ap-computer-science-principles/shared/assets/course-card.gif",
+        title: "Intro to Python",
+        description: "",
+        tags: ["Grades 7-10", "Python"],
+        gradeBand: "Middle-High School",
+        projectName: "",
+        projectType: "",
         video:
-          "https://res.cloudinary.com/dnx5si27b/video/upload/v1619531484/coverflow/Chess_Game_jhkugu.mp4",
+          "",
         standards: [
-          "AP CS A: MOD-1.A.1",
-          "AP CS A: VAR-2.A.1",
-          "AP CS A: VAR-3.A.1",
-          "AP CS A: CON-1.B.1",
+          "",
         ],
       },
       {
         image:
-          "https://uploads-ssl.webflow.com/6047d44fad4ed2d97b3a7a18/605162f8fa2a111e11dae7c4_AP%20CS%20Principles-min.gif",
-        title: "AP Computer Science Principles",
+          "https://uploads-ssl.webflow.com/6047d44fad4ed2d97b3a7a18/605162f60e5dc1824b0268e9_DSAI.gif",
+        title: "Data Science & Artificial Intelligence",
         description:
-          "Students take on the role of Cyber Crime Special Agents to solve criminal cases of stolen identity, trespassing, and malware attacks. A series of clues and background information are uncovered to assist with each thrilling investigation.",
-        tags: ["Grades 9-12", "Javascript"],
+          "",
+        tags: ["Grades 8-12", "Python"],
```

Comment:
Against my own instinct, but the source ticket suggests Grade 8-10 here

Reply:
Great catch, thank you!